manifests/service.pp

```diff
@@ -0,0 +1,40 @@
+# Class: shibboleth_idp::service
+#
+# Manages the Jetty process
+#
+# Sample Usage:
+#
+#    sometype { 'foo':
+#      notify => Class['shibboleth_idp::service'],
+#    }
+#
+#
+class shibboleth_idp::service {
+  # The base class must be included first because parameter defaults depend on it
+  if ! defined(Class['shibboleth_idp::params']) {
+    fail('You must include the shibboleth_idp::params class before using any shibboleth_idp defined resources')
+  }
+  validate_bool($shibboleth_idp::params::service_enable)
+  validate_bool($shibboleth_idp::params::service_manage)
+
+  case $shibboleth_idp::params::service_ensure {
```

Comment:
Can you switch all your vars to `$shibboleth_idp::service_ensure` for example.
Reading them from the init class allows them top be overwritten. They way it is now, they can not be set in init.